openshift/kustomize/tekton/base/tasks/buildah.yaml

```diff
@@ -65,8 +65,8 @@ spec:
         #!/usr/bin/env bash
         set -xe
 
-        export TMPDIR="/data/tmp/buildah"
-        mkdir -p "${TMPDIR}"
+        # export TMPDIR="$(workspaces.source.path)/tmp/buildah"
```

Comment:
Using the PVC results in a permission error.  I don't know how to fix this, so I'm commenting out for now.